radicle/src/cob/op.rs

```diff
@@ -182,3 +226,16 @@ impl<G: Signer, A: Clone> Actor<G, A> {
         op
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_opid_try_from_str() {
+        let str = "z6MksFqXN3Yhqk8pTJdUGLwATkRfQvwZXPqR2qMEhbS9wzpT/12";
+
+        let id = OpId::try_from(str).expect("Op ID parses string");
+        assert_eq!(str, id.to_string(), "string conversion is consistent");
+    }
```

Comment:
Would make sense to test the unhappy paths too :)

Reply:
Hmmm on it

Reply:
https://github.com/radicle-dev/heartwood/pull/255/commits/65f58353218fa571b36263dd106a96656f0c153c